src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -0,0 +1,127 @@
+            : base(Contracts.CheckRef(env, nameof(env)).Register(nameof(WordTokenizer)), MakeTransformer(env, columns, advancedSettings))
+        {
+        }
+
+        private static TransformWrapper MakeTransformer(IHostEnvironment env, (string input, string output)[] columns, Action<DelimitedTokenizeTransform.Arguments> advancedSettings)
+        {
+            Contracts.AssertValue(env);
+            env.CheckNonEmpty(columns, nameof(columns));
+            env.CheckValueOrNull(advancedSettings);
+            foreach (var (input, output) in columns)
+            {
+                env.CheckValue(input, nameof(input));
+                env.CheckValue(output, nameof(input));
+            }
+
+            // Create arguments.
+            var args = new DelimitedTokenizeTransform.Arguments
+            {
+                Column = columns.Select(x => new DelimitedTokenizeTransform.Column { Source = x.input, Name = x.output }).ToArray()
+            };
+            advancedSettings?.Invoke(args);
+
+            // Create a valid instance of data.
+            var schema = new SimpleSchema(env, columns.Select(x => new KeyValuePair<string, ColumnType>(x.input, TextType.Instance)).ToArray());
+            var emptyData = new EmptyDataView(env, schema);
+
+            return new TransformWrapper(env, new DelimitedTokenizeTransform(env, args, emptyData));
+        }
+    }
+
+    /// <summary>
+    /// Character tokenizer splits text into sequences of characters using a sliding window.
+    /// </summary>
+    public sealed class CharacterTokenizer: TrivialWrapperEstimator
+    {
+        /// <summary>
+        /// Tokenize incoming text in <paramref name="inputColumn"/> and output the tokens as <paramref name="outputColumn"/>.
+        /// </summary>
+        /// <param name="env">The environment.</param>
+        /// <param name="inputColumn">The column containing text to tokenize.</param>
+        /// <param name="outputColumn">The column containing output tokens. Null means <paramref name="inputColumn"/> is replaced.</param>
+        /// <param name="advancedSettings">Any advanced settings to be applied.</param>
+        public CharacterTokenizer(IHostEnvironment env, string inputColumn, string outputColumn = null,
+            Action<CharTokenizeTransform.Arguments> advancedSettings = null)
+            : this(env, new[] { (inputColumn, outputColumn ?? inputColumn) }, advancedSettings)
+        {
+        }
+
+        /// <summary>
+        /// Tokenize incoming text in input columns and output the tokens as output columns.
+        /// </summary>
+        /// <param name="env">The environment.</param>
+        /// <param name="columns">Pairs of columns to run the tokenization on.</param>
+        /// <param name="advancedSettings">Any advanced settings to be applied.</param>
+        public CharacterTokenizer(IHostEnvironment env, (string input, string output)[] columns,
```

Comment:
this is the pattern for the transformer conversion so far.
`params` is not possible here because of the `useMarkerCharacters`

---
In reply to: [218246352](https://github.com/dotnet/machinelearning/pull/931#discussion_r218246352) [](ancestors = 218246352)